clients/da-vinci-client/src/main/java/com/linkedin/davinci/replication/merge/MergeConflictResolverFactory.java

```diff
@@ -22,7 +22,7 @@ public MergeConflictResolver createMergeConflictResolver(
       ReadOnlySchemaRepository schemaRepository,
       RmdSerDe rmdSerDe,
       String storeName) {
-    return createMergeConflictResolver(schemaRepository, rmdSerDe, storeName, false);
```

Comment:
If we want to change the hardcode to true, we should also remove the throw exception command in the below and once we believe this is OK, we can create our own AA partial update store to test it before we announce "ready to serve".

Reply:
Done.